include/clang/StaticAnalyzer/Core/AnalyzerOptions.h

```diff
@@ -186,6 +186,10 @@ class AnalyzerOptions : public RefCountedBase<AnalyzerOptions> {
   // Cap the stack depth at 4 calls (5 stack frames, base + 4 calls).
   unsigned InlineMaxStackDepth = 5;
 
+  /// The maximal amount of translation units that is considered for import
+  /// when inlining functions during CTU analysis.
+  unsigned CTUImportThreshold{100u};
```

Comment:
I am not sure that aggregate initialization will be compiled without problems on GCC 4.8. Clang master still supports that compiler with C++11, right?
Perhaps ` = 100u;` would be more appropriate.